src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -2,12 +2,12 @@
 import * as React from 'react';
 import {Text, View, StyleSheet, TouchableOpacity, Pressable, Platform} from 'react-native';
 import ToastMessage from 'react-native-toast-message';
-import TextAreaChat from '../../../components/TextAreaChat';
-import ToggleSwitch from '../../../components/ToggleSwitch';
+import CheckBox from '@react-native-community/checkbox';
 import {profileSettingsDMpermission} from '../../../service/profile';
 import {addDotAndRemoveNewline} from '../../../utils/string/TrimString';
 import {COLORS} from '../../../utils/theme';
-import {useDynamicColors} from '../../../hooks/useToggleColors';
+import {PencilIcon} from '../../../assets';
+import {Divider} from '../../../components/Divider';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `CheckBox` from `@react-native-community/checkbox` is a good addition as it provides a native checkbox component for both iOS and Android. However, please ensure that the library version you're using doesn't have any known security vulnerabilities or bugs.



<!-- This is an auto-generated comment by OSS CodeRabbit -->